### PR TITLE
Closes #438 | Add dataloader for ASR-INDOCSC

### LINKED ISSUE #438: Create dataset loader for ASR-INDOCSC: AN INDONESIAN CONVERSATIONAL SPEECH CORPUS
Dataloader name: `asr_indocsc/asr_indocsc.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?asr_indocsc

|     Dataset| asr_indocsc |
|-------------|---|
| Description | This open-source dataset consists of 4.54 hours of transcribed Indonesian conversational speech on certain topics, where seven conversations between two pairs of speakers were contained. |
| Subsets | - |
| Languages | ind |
| Tasks | Text-To-Speech Synthesis, Automatic Speech Recognition |
| License | Creative Commons Attribution Non Commercial No Derivatives 4.0 (cc-by-nc-nd-4.0) |
| Homepage | https://magichub.com/datasets/indonesian-conversational-speech-corpus/ |
| HF URL | - |
| Paper URL | - |

### PR DESCRIPTION
**Pull request by zwenyu**
Closes #438 

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/{my_dataset}/{my_dataset}.py` (please use only lowercase and underscore for dataset folder naming, as mentioned in dataset issue) and its `__init__.py` within `{my_dataset}` folder.
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_LOCAL`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py` or `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py --subset_id {subset_name_without_source_or_seacrowd_suffix}`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Review by ljvmiranda921 — APPROVED** [review]
LGTM! Data loader works, thanks for the PR @zwenyu :)

**Comment by holylovenia** [comment]
A friendly reminder for @khelli07 to review.

**Comment by khelli07** [comment]
Hi, apparently the data can not be downloaded from my side. Seems like the server is down (?) or the pre-signed URL is invalid (?) 
image

I have already signed up and signed in to the platform. I have questions though. Do we have to log in to download? If yes, please include it in the description.

**Comment by ljvmiranda921** [comment]
Oh wow that's interesting. It did work the last time I checked this. Hmm, I guess we should make the file `LOCAL` instead?

**Comment by zwenyu** [comment]
@khelli07 Yes need to create an account and be logged in, I'll add this in the description. Are you able to download the dataset by running the dataloader script? I'm now getting the same access error when trying to download through [here](https://magichub.com/df/df.php?file_name=Indonesian_Conversational_Speech_Corpus.zip) or manually through [the dataset landing page](https://magichub.com/datasets/indonesian-conversational-speech-corpus/), but is still able to download and load the dataset through the script. Tried this after deleting the previous cached copy.

**Comment by holylovenia** [comment]
> @khelli07 Yes need to create an account and be logged in, I'll add this in the description. Are you able to download the dataset by running the dataloader script? I'm now getting the same access error when trying to download through [here](https://magichub.com/df/df.php?file_name=Indonesian_Conversational_Speech_Corpus.zip) or manually through [the dataset landing page](https://magichub.com/datasets/indonesian-conversational-speech-corpus/), but is still able to download and load the dataset through the script. Tried this after deleting the previous cached copy.

I managed to download the data from [here](https://magichub.com/df/df.php?file_name=Indonesian_Conversational_Speech_Corpus.zip), @zwenyu. Probably it's a temporary issue?

cc: @khelli07 @ljvmiranda921

**Comment by khelli07** [comment]
Just ran it again, and it works! Let me take a look a bit more and wrap up the review :)

File: seacrowd/sea_datasets/asr_indocsc/asr_indocsc.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+# no bibtex citation
+_CITATION = ""
+_DATASETNAME = "asr_indocsc"
+_DESCRIPTION = """\
+This open-source dataset consists of 4.54 hours of transcribed Indonesian
+conversational speech on certain topics, where seven conversations between two
+pairs of speakers were contained.
```
Comment (khelli07) [inline seacrowd/sea_datasets/asr_indocsc/asr_indocsc.py:33]: The rest looks good to me! Just need to add a description that you have to sign up and/or login first before running this. Thanks! 😃

Comment (zwenyu) [reply seacrowd/sea_datasets/asr_indocsc/asr_indocsc.py]: Thanks for checking. I've added the instructions and pushed the change.

**Pull request merged**